install/linux/docker-ce/ubuntu.md

```diff
@@ -120,11 +120,10 @@ from the repository.
 
     ```bash
     $ sudo apt-key fingerprint 0EBFCD88
-
```

Comment:
I think the empty line (although not actually there when you run the command) was added in the documentation to better separate the command to run, and the output it produces.

Would be good to add it back, but definitely not a show-stopper 🤗

Reply:
Agreed. I'll add it back and merge.